src/unix/fsevents.c

```diff
@@ -267,12 +271,6 @@ static void uv__fsevents_event_cb(ConstFSEventStreamRef streamRef,
         }
       }
 
-#ifdef MAC_OS_X_VERSION_10_7
-      /* Ignore events with path equal to directory itself */
-      if (len == 0)
```

Comment:
I'm still learning the codebase. Any tips on checking if the handle represents a file vs a directory here?